src/lib/forward.ts

```diff
@@ -95,8 +112,14 @@ export class EventForwarder {
 
     node.on("ready", (changedNode: ZWaveNode) => {
       // Dump full node state on ready event
-      const nodeState = dumpNode(changedNode);
-      notifyNode(changedNode, "ready", { nodeState });
+      this.clients.clients.forEach((client) =>
+        this.sendEvent(client, {
```

Comment:
We no longer use `notifyNode` here.

Reply:
Correct, is no longer needed. notifyNode was a helper